### DISCUSSION TITLE
GDPR?

### DISCUSSION THREAD

**Discussion by monodera** [Subaru-PFS/spt_target_uploader#159] [General]
Do we need to show GDPR compliant message? I'm not quite sure.

**Comment by yukimoritani** [comment]
I'm not sure.. but do we ask any items relating to users privacy information? Or are we able to identify users?

**Reply by monodera** [reply]
Google Web Font is used and it appears to violate GDPR. 

MkDocs which is used to build the documentation has a privacy module, but Panel for the web app does not. One possible resolution is to download the font file locally and serve it, which is an acceptable solution, I guess.

**Comment by monodera** [comment]
How to setup in MkDocs: https://squidfunk.github.io/mkdocs-material/setup/ensuring-data-privacy/#cookie-consent-custom-cookie

**Comment by monodera** [comment]
I found a news article below. 

**Europe’s cookie nightmare is crumbling**

The European Commission wants browsers to manage cookie preferences instead of pop-ups on every website.

https://www.theverge.com/news/823788/europe-cookie-prompt-browser-changes-proposal
